utils/.config.template.toml

```diff
@@ -25,16 +25,16 @@ storymode = { optional = true, type = "bool", default = false, example = false,
 
 
 [settings.background]
-background_choice = { optional = true, default = "minecraft", example = "rocket-league", options = ["minecraft", "gta", "rocket-league", "motor-gta", "csgo-surf", "cluster-truck", ""], explanation = "Sets the background for the video based on game name" }
+background_choice = { optional = true, default = "minecraft", example = "rocket-league", options = ["", "minecraft", "gta", "rocket-league", "motor-gta", "csgo-surf", "cluster-truck"], explanation = "Sets the background for the video based on game name" }
```

Comment:
Empty chose a random one keep that in

Reply:
@JasonLovesDoggo I moved it to the beginning of the options because when the user adds background it is appended to the end of the list, so I thought that it might be better to keep the random option as the first one, separated from other backgrounds. I also skip it in in jinja template: `{% for background in checks["background_choice"]["options"][1:] %}`. I can try to make it work by placing a random option at the end, but I think it's better to leave it at the beginning. It also works the same.

Reply:
No, no that looks fine It's just 5:00 a.m. so I kind of didn't read the code properly